src/legacy/core_plugins/data/public/expressions/expressions_service.test.tsx

```diff
@@ -53,7 +53,7 @@ describe('expressions_service', () => {
   let testAst: Ast;
 
   beforeEach(() => {
-    interpretAstMock = jest.fn(_ => Promise.resolve(expressionResult));
+    interpretAstMock = jest.fn((_, __, ___) => Promise.resolve(expressionResult));
```

Comment:
Not relevant for the review, but could you explain why this is necessary for TS vs. not passing any arguments?

```ts
interpretAstMock = jest.fn(() => Promise.resolve(expressionResult));
```

Reply:
In the new TS version, a function that doesn't take an arguments doesn't match the type restriction of a function that takes three arguments. Could have also done `(..._) => ...` I'd guess, this was just the first thing I tried.